cmd/proxy/actions/home_test.go

```diff
@@ -0,0 +1,30 @@
+package actions
+
+import (
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestProxyHomeHandler(t *testing.T) {
```

Comment:
The same for this test also, the benefit from this test is not a lot imo. If you can convince me otherwise, that will be sweet.

Reply:
Agree with @manugupt1 can we remove this test and keep the basic auth one. I'll merge afterwards. Thanks @PascalDierich

Reply:
Removed the handler tests. @marwan-at-work @manugupt1 thanks for your input 👍